SSL Certificate identification not working on MacOS

The following lines are failing on MacOS:

```
try:
    os.path.isfile('/etc/ssl/certs/ca-certificates.crt')
    CA_CERTS = '/etc/ssl/certs/ca-certificates.crt'
except Exception:
    import certifi
    CA_CERTS = certifi.where()
```

the expression `os.path.isfile` returns a bool and therefore the try/except clause is ineffective. On MacOS this results in the following error:

`[ERROR] network_handler(): HTTPSConnectionPool(host='www.alexblackie.com', port=443): Max retries exceeded with url: /articles/email-authenticity-dkim-spf-dmarc/?utm_source=hackernewsletter&utm_medium=email&utm_term=fav (Caused by SSLError(FileNotFoundError(2, 'No such file or directory')))`

--- a/buku
+++ b/buku
@@ -107,10 +107,9 @@ LOGERR = LOGGER.error
 # Define the default path to ca-certificates
 # In Linux distros with openssl, it is /etc/ssl/certs/ca-certificates.crt
 # Fall back to use `certifi` otherwise
-try:
-    os.path.isfile('/etc/ssl/certs/ca-certificates.crt')
+if sys.platform.startswith('linux') and os.path.isfile('/etc/ssl/certs/ca-certificates.crt'):
     CA_CERTS = '/etc/ssl/certs/ca-certificates.crt'
-except Exception:
+else:
     import certifi
     CA_CERTS = certifi.where()
 

